services/storage/src/simcore_service_storage/dsm.py

```diff
@@ -472,29 +473,23 @@ async def try_update_database_from_storage(
 
                     return to_meta_data_extended(row)
         except botocore.exceptions.ClientError:
-            if silence_exception:
-                logger.debug("Error happened while trying to access %s", file_uuid)
-            else:
-                logger.warning(
-                    "Error happened while trying to access %s", file_uuid, exc_info=True
-                )
-            # the file is not existing or some error happened
-            return None
+            logger.debug("Error happened while trying to access %s", file_uuid)
+            if reraise_exceptions:
+                raise
```

Comment:
if ``reraise_exceptions==False``, shouldn't we log it?

Reply:
no need it would get caught

Reply:
It is already logged above

Reply:
i meant e.g. as a warning